docs/README.md

```diff
@@ -2309,8 +2309,7 @@ You don't need to know anything about this to develop using Booster, neither in
 
 One of the goals of Booster is to become provider agnostic so you can deploy your application to any serverless provider like AWS, Google Cloud, Azure, etc...
 
-So far, in the current version, only AWS is supported, but given the high level of abstraction, it will eventually support
-all cloud providers. (**Contributions are welcome!** 😜)
+In the current version, we offer full support for AWS provider and experimental support for Kubernetes and Azure providers.
```

Comment:
I would add after that sentence that "we will eventually support all main cloud providers (contributions are welcome!)" because GCP will be implemented in the future